templates/web/bexley/waste/services.html

```diff
@@ -1,16 +1,28 @@
 [% PROCESS 'waste/_services_header.html' %]
 [% PROCESS 'waste/_service_missed.html' %]
 
-[% IF unit.requests_open %]
+[% IF unit.delivery_open %]
   <span class="waste-service-descriptor">
-    A [% unit.service_name FILTER lower %] container request has been made. Check status below:
+    A [% unit.service_name FILTER lower %] container delivery request has been made. Check status below:
     <ul>
-      <li>Reported on: [% unit.request_details.reported ? date.format(unit.request_details.reported) : 'N/A' %]</li>
-      <li>Will be completed by: [% unit.request_details.will_be_completed ? date.format(unit.request_details.will_be_completed) : 'N/A' %]</li>
-      <li>Action: [% unit.request_details.latest_comment %]</li>
+      <li>Reported on: [% unit.delivery_details.reported ? date.format(unit.delivery_details.reported) : 'N/A' %]</li>
+      <li>Will be completed by: [% unit.delivery_details.will_be_completed ? date.format(unit.delivery_details.will_be_completed) : 'N/A' %]</li>
+      <li>Action: [% unit.delivery_details.latest_comment %]</li>
     </ul>
   </span>
+[% END %]
+
+[% IF unit.removal_open %]
+  <span class="waste-service-descriptor">
+    A [% unit.service_name FILTER lower %] container collection request has been made. Check status below:
+    <ul>
+      <li>Reported on: [% unit.removal_details.reported ? date.format(unit.removal_details.reported) : 'N/A' %]</li>
+      <li>Will be completed by: [% unit.removal_details.will_be_completed ? date.format(unit.removal_details.will_be_completed) : 'N/A' %]</li>
+      <li>Action: [% unit.removal_details.latest_comment %]</li>
+    </ul>
+  </span>
+[% END %]
 
-[% ELSIF unit.delivery_allowed || unit.removal_allowed %]
+[% IF unit.delivery_allowed || unit.removal_allowed %]
```

Comment:
Hopefully this is the links under each container, let me know if I've missed somewhere :)

Reply:
I've added checks for open reports in templates/web/bexley/waste/_services_request.html, which seems to work, will push a little fix though it might be good to have some tests. (Also, my 'delivery_allowed' etc. naming is confusing, given that that only means 'you can make a delivery request on this container assuming you don't have open requests etc.)

Reply:
Thanks. It matches the request_allowed in Echo (which also is separate from open requests), so that's fine, I think.